src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/OpphørValideringService.kt

```diff
@@ -47,6 +48,49 @@ class OpphørValideringService(
         }
     }
 
+    fun validerIngenUtbetalingEtterRevurderFraDatoLæremidler(
```

Comment:
Tror ikke jeg helt skjønte behovet bak metoden her 👀
Validerer vi at `beregningsresultatForOpphør` fungerer her?

Reply:
Tanken her er at man skal validere at det ikke går noen utbetalinger etter RevurderFra-dato, på samme måte som for TilsynBarn: https://github.com/navikt/tilleggsstonader-sak/pull/561/files#diff-5c4e5bf48755a7dc74e2034da1fab51e31908b05450539ef45b1b00c13f949adR36

Reply:
Fjernet: b1decb88c97731f811d4d6f76560df15412f9856